pkg/ingester/ingester.go

```diff
@@ -51,14 +51,16 @@ var (
 		Help: "The total number of queries the ingester has handled.",
 	})
 	queriedSamples = promauto.NewHistogram(prometheus.HistogramOpts{
-		Name:    "cortex_ingester_queried_samples",
-		Help:    "The total number of samples returned from queries.",
-		Buckets: prometheus.DefBuckets,
+		Name: "cortex_ingester_queried_samples",
+		Help: "The total number of samples returned from queries.",
+		// Could easily return 1m samples per query - 10*(8^9) = 1.3m.
+		Buckets: prometheus.ExponentialBuckets(10, 8, 9),
```

Comment:
Could I ask you to bump this up by one more factor? Or maybe start at 80 instead of 10? I have queries that can pull back more than 1.3m samples

Reply:
Can do!  Once we merge the ingester streaming one its less of an issue.

Also, this is per-ingester; not a global query limit.